test_projects/ios/EarlGreyExample/ops.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+EARL_GREY_EXAMPLE="$TEST_PROJECTS_IOS/EarlGreyExample"
+
+function setup_ios_env() {
+  if ! [ -x "$(command -v xcpretty)" ]; then
+    gem install cocoapods
```

Comment:
Can we use:
`gem install cocoapods -v 1.9.3`
This gem is getting updated pretty often.